index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please fix the linter manually instead of disabling them. It will help you to write better code.

Reply:
Thanks, I've just done it